ansible/roles/xsnippet_api/templates/caddy.j2

```diff
@@ -0,0 +1,28 @@
+(cors) {
+  @cors_preflight method OPTIONS
+  @cors header Origin {args.0}
```

Comment:
@malor This is a matching query. CORS headers are returned only if a request contains `Origin` header that matches expected one (i.e. `xsnippet-web`). I.e. CORS headers are returned for browsers and won't be returned for pure API requests.